eng/pipelines/runtime.yml

```diff
@@ -534,15 +524,24 @@ jobs:
     buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
     runtimeFlavor: mono
     platforms:
-      - tvOS_x64
-      - tvOS_arm64
-      - iOS_x64
-      # - iOS_arm # https://github.com/dotnet/runtime/issues/34465
-      - iOS_arm64
-      - Android_x64
-      - Android_x86
-      - Android_arm
-      - Android_arm64
+    - Android_x64
+    - Android_arm
+    - tvOS_arm64
+    # - iOS_arm # https://github.com/dotnet/runtime/issues/34465
+    - iOS_x64
+    jobParameters:
+      liveRuntimeBuildConfig: release
+      liveLibrariesBuildConfig: Release
```

Comment:
Nit: why the casing differences here in release?

Reply:
It’s inconsistency we have in between subsets. Installer and Libraries take uppercase and the artifacts it produce are uppercase, vs runtimes (mono and coreclr) produce lowercase. It’s something we should fix.